react/react1/week2/to-do/src/component/ToDoList.jsx

```diff
@@ -0,0 +1,54 @@
+import { React } from "react";
+import { useState } from "react";
+import ToDoDb from "./ToDoDb.js";
+import ToDoHeader from "./Header";
+import TodoItem from "./TodoItem";
+const ToDo = () => {
+  const [todos, setToDos] = useState(ToDoDb);
+  const [newToDo, setNewToDo] = useState({});
+  const NewToDo = (description) => {
+    setNewToDo({
+      id: Math.floor(Math.random() * (100 - 4 + 1) + 4),
```

Comment:
After some iterations, random might generate the same number again then it will throw an error!
Research for unique id's For example `nanoid`

Reply:
Yes ,I havr same thought ,you are correct,I will do this and thank you very much for review.

Reply:
Thanks for your time and review.